src/objects/zcl_abapgit_object_clas.clas.abap

```diff
@@ -330,6 +330,11 @@ CLASS ZCL_ABAPGIT_OBJECT_CLAS IMPLEMENTATION.
                       iv_package = iv_package ).
 
     deserialize_docu( io_xml ).
+
+    zcl_abapgit_objects_activation=>add(
```

Comment:
this should not be needed, and not related to the original issue?

Reply:
It's moved from ZCL_ABAPGIT_OO_CLASS, I was influenced by "TODO, perhaps move this call to somewhere else, to be done while cleaning up the CLAS deserialization", but I realize that there's also another such an activation for REPT still in ZCL_ABAPGIT_OO_CLASS. Okay to keep it at its original place.